README.md

```diff
@@ -53,9 +53,18 @@ When installed via npm, videojs-playlist supports Browserify-based workflows out
 For full details on how to use the playlist plugin can be found in [the API documentation](docs/api.md).
 
 ```js
-var player = videojs('video');
-
-player.playlist([{
+// Initialize Video.js player
+const player = videojs('video');
+
+// Initialize the playlist plugin
+const playlistPlugin = player.playlist({
+  // Play through the playlist automatically with no delay between videos
+  autoadvanceDelay: 0,
+  // Loop the playlist back to the beginning after the last video 
+  repeat: true
+});
```

Comment:
do we need these options if we have `setAutoadvanceDelay` and methods to toggle `repeat`?